src/tags/everyone.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @Author: stupid cat
+ * @Date: 2017-05-07 18:38:55
+ * @Last Modified by: stupid cat
+ * @Last Modified time: 2017-10-19 18:12:29
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
+
+const Builder = require('../structures/TagBuilder');
+
+module.exports =
+    Builder.BotTag('everyone')
+        .withArgs(a => [a.optional('enabled')])
+        .withDesc(
+            'Returns the mention of `@everyone`.\n' +
+            'If `enabled` is `false` it will return a non-pinging `@everyone`, defaults to `true`'
+        )
+        .withExample(
+            '{everyone}',
```

Comment:
If updating the subtag's name (and adding an alias), make sure you also not use the alias but the real subtag name in the example.
```js
.withExample(
  '{everyonemention}', 
  '@everyone'
)
```